frontend/src/pages/Agency/Agency.js

```diff
@@ -82,122 +100,135 @@ function AgencyTable() {
     .catch(err => {
       console.log(err);
     });
-  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   /**
-   * main filtering method
+   * Main filter function that passes the data for each agency (each row in the table) to a helper filtering method
+   * @param {Array} rows The unfiltered data that is an array of each row of the table, i.e. each
+   * entry of the array contains the information corresponding to each agency in the table
    */
   function search(rows) {
     return rows.filter(
       (row) => 
-        checkOptions(row, filters)
+        checkOptions(row)
     );
   };
 
-
-/**
- * Perform search then call filter method to filter based on options
- */
-function checkOptions(row, filters){
-  for(let option in filters){
-    //perform search
-    if(option === "search") {
-      let found = false;
-      //search based on each word in the name
-      let words = row.tableContent.name.toLowerCase().split(' ');
-      let searched = filters.search.toLowerCase();
-      for(let word of words){
-        found = word.startsWith(searched);
-        if(found){
-          break;
+  /**
+   * Filters the current row of the table based on the options set in the filters object. First performs the search based on
+   * the currently set search query. Then, the row is filtered based on each dropdown option by calling the helper checkStatuses
+   * @param {Object} row JSON object that contains the data for the current row being checked against the set filter options
+   * @returns Boolean value, true if the current row's data matches the filter options and false otherwise
+   */
+  function checkOptions(row){
```

Comment:
For styling consistency, please add spaces in between text and the opening brackets, for example `function checkOptions(row){` should be `function checkOptions(row) {` and so on. Please see if you can do so for this file and anywhere else that may need it!